### PR TITLE
Update WM.yaml

### PR DESCRIPTION
**Pull request by Karthi8883**
added water_linearvelocity_sensor point

### CODE REVIEW

**Comment by tasodorff** [comment]
validator errors

File: ontology/yaml/resources/METERS/entity_types/WM.yaml [hunk 1]
```diff
@@ -31,4 +31,7 @@ WM_WF:
   - WM
   is_canonical: true
   uses:
-  - supply_water_flowrate_sensor
\ No newline at end of file
+  - supply_water_flowrate_sensor
+  opt_uses:
+  - water_linearvelocity_sensor
```
Comment (tasodorff) [inline ontology/yaml/resources/METERS/entity_types/WM.yaml:36]: if we are being specific about it being supply, then please add supply

**Review by tasodorff — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/METERS/entity_types/WM.yaml (hunk 1 above)
Comment (Karthi8883) [reply ontology/yaml/resources/METERS/entity_types/WM.yaml:36]: We are not specific about it being supply - prefer it to be generic but the supply_water_flowrate_sensor field was already existing and did not want to break existing model and hence added new fields in the opt_uses.

**Review by tasodorff — APPROVED** [review]
Opening issue to remove `supply` from this definition.

**Comment by mschulze17** [comment]
@cstirdivant yours to merge

**Pull request merged**